installers/auth.example.org/operations/petstore/order/inventory.edn

```diff
@@ -0,0 +1,38 @@
+{:install
+ {:juxt.site/subject-uri "https://auth.example.org/_site/subjects/system"
+  :juxt.site/operation-uri "https://auth.example.org/_site/operations/create-operation"
+  :juxt.site/input
+  {:xt/id "{{$id}}"
+   :juxt.site/do-operation-tx-fn "https://auth.example.org/_site/do-operation"
+
+   :juxt.site/transact
+   {:juxt.site.sci/program
+    #juxt.pprint
+     {:ring.response/body
+      (->>
+       (xt/q
+        '{:find [(pull e [*])]
+          :where [[e :juxt.site/type "https://data.example.org/types/order"]]})
```

Comment:
this is the list of orders, right? is this what's expected from inventory?

Reply:
Strange, given that it's called inventory I'd expect it to be metrics of the inventory (IE pets), rather than a query on order log.

Reply:
I do agree, but that's what the reference implementation does. Plus in that case, it would be under `/pets` rather than `/orders`